### PR TITLE
Update appimage.rst

### PR DESCRIPTION
**Pull request by anki-code**
Hello! Since we have AppImage in the Github assets I improved the doc. Thanks!

### CODE REVIEW

File: docs/appimage.rst [hunk 1]
```diff
@@ -3,14 +3,40 @@ Via AppImage
 
 `AppImage <https://appimage.org/>`_ is a format for distributing portable software on Linux without needing superuser permissions to install the application. It tries also to allow Linux distribution-agnostic binary software deployment for application developers, also called Upstream packaging. 
 
-AppImage allows xonsh to be run on any AppImage supported Linux distributive without installation and root access.
+In short the AppImage is the one executable file which contains the xonsh and Python. AppImage allows xonsh to be run on any AppImage supported Linux distributives without installation and root access.
```
Comment (scopatz) [inline docs/appimage.rst:6]: `In short the AppImage is one executable file which contains both xonsh and Python. AppImage allows xonsh to be run on any AppImage supported Linux distribution without installation or root access.`

File: docs/appimage.rst [hunk 2]
```diff
@@ -3,14 +3,40 @@ Via AppImage
...
+
+Get AppImage from Github
+------------------------
+You can get xonsh AppImage from Github and run it on your Linux without install:
```
Comment (scopatz) [inline docs/appimage.rst:10]: You can get the xonsh AppImage from GitHub and run it on your Linux machine without installing it:

File: docs/appimage.rst [hunk 3]
```diff
@@ -3,14 +3,40 @@ Via AppImage
...
+
+.. code-block:: bash
+
+    wget https://github.com/xonsh/xonsh/releases/latest/download/xonsh-x86_64.AppImage -O xonsh
+    chmod +x xonsh
+    ./xonsh
+
+If you haven't Python on your host you may want to get it from AppImage:
```
Comment (scopatz) [inline docs/appimage.rst:18]: If you don't have Python on your host, you may want to get it from AppImage:

File: docs/appimage.rst (hunk 1 above)
Comment (anki-code) [reply docs/appimage.rst:6]: Thanks!

File: docs/appimage.rst (hunk 3 above)
Comment (anki-code) [reply docs/appimage.rst:18]: Thanks!

File: docs/appimage.rst (hunk 2 above)
Comment (anki-code) [reply docs/appimage.rst:10]: Thanks!

**Comment by scopatz** [comment]
Thanks @anki-code!

**Pull request merged**